assets/app/scripts/app.js

```diff
@@ -122,6 +122,10 @@ angular
         templateUrl: 'views/browse/build-config.html',
         controller: 'BuildConfigController'
       })
+      .when('/project/:project/browse/builds/:buildconfig/edit', {
```

Comment:
I like the URL, but it would conflict with the route below if we ever had a build named `edit`.